fs/src/jvmMain/kotlin/com/suparnatural/core/fs/FileSystem.kt

```diff
@@ -0,0 +1,228 @@
+package com.suparnatural.core.fs
+
+import java.io.*
+import java.nio.file.Paths
+import java.util.*
+
+actual object FileSystem {
+
+    actual val contentsDirectory: Path
```

Comment:
It would be better if we leave contents, temporary and caches directories as empty (Path object with relative and absolute values as null) as they don't have any significance on JVM. Also, it may be confusing because depending on the platform, the cache directory will change. For example cache on mac os vs cache on windows.

Reply:
I agree the directories have no default equivalent on the jvm. The directories are used in the common tests therefore they would need to be set for the tests. I will adjust the prepare function of test accordingly.

Reply:
I set the directories to empty paths on JVM and adjusted the tests accordingly in the latest commit.